### ISSUE TITLE
Switching to a different scene collection with NDI Audio Output causes crash

### ISSUE THREAD

**Issue by Oppzippy** [Palakis/obs-ndi#236]
To reproduce the issue, create a new scene collection and add the Dedicated NDI output (audio only) filter to any audio track. Now switch scene collections, and that should cause the crash.

OBS crash report: https://obsproject.com/logs/APYsKdQLR3EmlsHm

**Comment by GuestJim** [comment]
I'm glad I found this open issue as I have been running into this too. When I removed the audio-only NDI output filters, OBS did not crash when I tried switching scene collection, and then did once I restored the filters.
Here is a crash report and a log file:
[Crash Report](https://obsproject.com/logs/N-s77Ui3cyfg3VUQ)
[Log File](https://obsproject.com/logs/lkk6yTKdrfavNRrH)
Until this is fixed, I guess I will be editing the global.ini file to swap scene collections.
